controllers/numaresourcesoperator_controller.go

```diff
@@ -388,7 +388,7 @@ func (r *NUMAResourcesOperatorReconciler) deleteUnusedDaemonSets(ctx context.Con
 					klog.ErrorS(err, "error while deleting daemonset", "DaemonSet", ds.Name)
 					errors = append(errors, err)
 				} else {
-					klog.V(3).Infof("Daemonset [%s] deleted", ds.Name)
+					klog.V(3).InfoS("Daemonset deleted", "name", ds.Name)
```

Comment:
I should have explained better in the commit message. This is to make the output more consistent when we share the logging backend.

Reply:
> share the logging backend.

Sorry, but do you mind elaborating?

Reply:
loggers can be quite simple (the stdlib log package) or multi layered components. `klog` and `zap` are build both on top of interfaces - present in the stdlib or emerged from the community. `zap` is much more complex and composable, yet fast.
The operator framework suggests `zap`, but we moved to `klog` because we like it more. This meant we have two almost independent logging stacks in our codebase, up until https://github.com/openshift-kni/numaresources-operator/pull/363/commits/7b51ff0af2622a62c0598499a74191be695fdb6a which unifies the (improper naming) `backend` layer, aka the way messages are actually logged out. There's no real reason to keep two fully independent stacks, and sharing the backend is totally fair game - we just didn't manage to get there yet.
This also ties in into `Infof` vs `InfoS`, with the shared zap-based backend `InfoS` is logged with a nicer message.

Reply:
Thank you for the comprehensive explanation